spec/controllers/openid_application_controller_spec.rb

```diff
@@ -258,7 +258,7 @@ def index
 
       it 'should return 200 and add user to session' do
         with_okta_configured do
-          Okta::Service.any_instance.stub(:user).and_return(faraday_response)
+          allow_any_instance_of(Okta::Service).to receive(:user).and_return(faraday_response)
           allow(faraday_response).to receive(:success?).and_return(true)
           allow(faraday_response).to receive(:body).and_return(okta_response)
```

Comment:
possibly consider using VCR cassettes instead of stubbing / mocking? unless there is challenges to that of course, as is often the case with saml and encrypted payloads and such because of the need to use Timecop and override Kernel.random and other things.